Homework1/Task03/task03.py

```diff
@@ -0,0 +1,20 @@
+"""
+Write down the function, which reads input line-by-line, and find maximum and minimum values.
+Function should return a tuple with the max and min values.
+For example for [1, 2, 3, 4, 5], function should return [1, 5]
+We guarantee, that file exists and contains line-delimited integers.
+To read file line-by-line you can use this snippet:
+with open("some_file.txt") as fi:
+    for line in fi:
+        ...
+"""
+from typing import Tuple
+
+
+def find_maximum_and_minimum(file_name: str) -> Tuple[int, int]:
```

Comment:
What is the memory complexity of this solution?   N is the number of rows in file.

Reply:
The memory complexity is N. Do you mean that I shouldn't create a list and compare values in a loop?

Reply:
yes, in this case this optimization looks reasonable.